src/lls_model.jl

```diff
@@ -311,6 +311,6 @@ function NLPModels.ghjvprod!(
   @lencheck nls.meta.nvar x g v
   @lencheck nls.meta.ncon gHv
   increment!(nls, :neval_hprod)
-  gHv .= zeros(T, nls.meta.ncon)
+  gHv .= zero(T)
```

Comment:
I change to this instead of `fill!(S(undef, nls.meta.ncon), zero(T))` I hope I am not mistaken

Reply:
This is fine